src/redux/user/asyncThunks.js

```diff
@@ -33,7 +33,7 @@ export const adminFetchUsers = createAsyncThunk(
   async (_, { dispatch, rejectWithValue }) => {
     try {
       const { status, data } = await fetchWithCreds({
-        path: `${userapiPath}admin/user`,
```

Comment:
Does the `userapiPath` not already have the right value here? I think we might need to just change `user` -> `get_users`?

Reply:
oh got it, userapiPath is `user/admin/user` here